bootstrap/terraform/vm.tf

```diff
@@ -111,7 +111,7 @@ resource "azurerm_public_ip" "vault_azure_pub_ip" {
 
 # Restrict SSH access to the local public IP
 data "http" "my_ip" {
-  url = "https://ifconfig.me/ip"
+  url = "https://ipinfo.io/ip"
```

Comment:
For my learning purpose, could you share we made this change?

Reply:
I was unable to ssh into the Azure VM. "https://ifconfig.me/ip" returns an IPv6 and switching to "https://ipinfo.io/ip" resolves the issue. It returns IPv4. Maybe the original "https://ifconfig.me/ip" endpoint changed its behavior at some point?